README.md

```diff
@@ -148,7 +149,17 @@ The typical workflow involves retrieving a PSD document manifest file via `/docu
 
 For the time being clients can only use assets stored on EITHER Adobe's Creative Cloud OR external storage (like AWS S3 or Azure Blog Storage).  Support for mixing and matching will be added in the future
 
-### Fonts
+### Text Layer
+
+The Photoshop APIs currently support creating and editing of Text Layer with different fonts, character styles and paragraph styles.
+
+Please see the api docs for more information.
+https://git.corp.adobe.com/pages/dice/pie-in-the-sky/#api-Photoshop-document_operations
```

Comment:
@khound link is wrong, it points to the private page

Reply:
@noahhoffman Yes. Thanks for pointing out.